docs/overview/timeline.md

```diff
@@ -164,7 +164,7 @@ var face_name_procedure = {
 ### Using in a function
 
 Continung the example from the previous section, what if we wanted to show the name with the face, combining the two variables together? 
-To do this, we can use a [dynamic parameter](dynamic-parameters) (a function) to create an HTML-string that uses both variables in a single parameter.
+To do this, we can use a [dynamic parameter](..overview/dynamic-parameters/) (a function) to create an HTML-string that uses both variables in a single parameter.
```

Comment:
I think the fix here is actually just to add .md to the link so that our docs builder interprets it as a link to the relevant document rather than a generic link. 

```suggestion
To do this, we can use a [dynamic parameter](dynamic-parameters.md) (a function) to create an HTML-string that uses both variables in a single parameter.
```

Reply:
good point! Thanks for the review 😊